### ISSUE TITLE
Install should simply be 'pip install goose-extractor'?

### ISSUE THREAD

**Issue by andybak** [grangier/python-goose#255]
The suggested install instructions don't mesh well with common workflows. I'm currently hitting an unrelated install problem so I can't verify this works but shouldn't the install instructions just be 'pip install'?

**Comment by vindarel** [comment]
Hi, there is `goose-extractor` on pypi ! 

Then `from goose import Goose`.

**Comment by andybak** [comment]
Yeah - I can't remember in detail now but I think the point I was making was that "it doesn't say that in the readme".
